src/windshaft/map-base.js

```diff
@@ -47,13 +47,21 @@ var WindshaftMap = Backbone.Model.extend({
   },
 
   // TODO: Move this somewhere else and keep this class as a wrapper for windshaft responses
-  createInstance: function (options) {
+  createInstance: function (options, includeFilters) {
```

Comment:
I'd treat `includeFilters` as an option here too... IMO it's more clear to have something like `wadus({ includeFilter: true });` than `wadus(true)`.